tests/space-time-stack/test_demangling.cpp

```diff
@@ -0,0 +1,90 @@
+#include <iostream>
+#include <regex>
+#include <sstream>
+
+#include "Kokkos_Core.hpp"
+#include "kp_all.hpp"
```

Comment:
What do you use from that header?

Reply:
I think it's a leftover. I removed `kp_all.hpp` :wastebasket: